pkg/cloudprovider/providers/azure/azure_storage.go

```diff
@@ -19,24 +19,25 @@ package azure
 import (
 	"fmt"
 
-	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2017-10-01/storage"
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2018-07-01/storage"
 	"github.com/golang/glog"
 )
 
 const (
 	defaultStorageAccountType      = string(storage.StandardLRS)
+	defaultStorageAccountKind      = storage.StorageV2
 	fileShareAccountNamePrefix     = "f"
 	sharedDiskAccountNamePrefix    = "ds"
 	dedicatedDiskAccountNamePrefix = "dd"
 )
 
 // CreateFileShare creates a file share, using a matching storage account
-func (az *Cloud) CreateFileShare(shareName, accountName, accountType, resourceGroup, location string, requestGiB int) (string, string, error) {
+func (az *Cloud) CreateFileShare(shareName, accountName, accountType, accountKind, resourceGroup, location string, requestGiB int) (string, string, error) {
```

Comment:
As a public method on a public object shouldn't we add a release note for this change?

Reply:
fixed comments in new commit, PTAL, thanks.